src/player/a11yHelpers.ts

```diff
@@ -0,0 +1,9 @@
+import * as vscode from "vscode";
+
+export function isAccessibilitySupportOn(): boolean {
+    const config = vscode.workspace.getConfiguration('editor');
+    // Possible values are: auto, on, off. 
+    // `auto` then queries whether the screenreader is actually on.
+    // For the purposes of this demo, `auto` is treated as `on`.
+    return config.get('accessibilitySupport') !== 'off';
```

Comment:
Do we plan to change this after the demo?

Reply:
I'm not sure, definitely open to discussion on that. I think the main risk of leaving it is like it is, is that we'd accidentally be including whatever extra accessibility decorations even if the screenreader was off... but that seems like a better alternative to me than accidentally not including them when the screenreader is on.